meinberlin/apps/contrib/templates/meinberlin_contrib/bo_component_library.html

```diff
@@ -0,0 +1,19 @@
+{% extends 'base.html' %}
+{% load static %}
+
+{% block title %}Component Library &mdash; {{ block.super }}{% endblock %}
+
+{% block content %}
+<div id="layout-grid" role="main">
+  <div id="layout-grid__area--leftcolumn">
+    <!-- LEFT COLUMN not used -->
+  </div>
+  <div id="layout-grid__area--maincontent">
+    {% include 'meinberlin_contrib/components/breadcrumb.html' %}
+  </div>
+  <!-- Component list and links -->
+  <div id="layout-grid__area--marginal" role="complementary" class="marginal">
+    <a href="#layout-grid__area--contentheader" class="">Breadcrumb</a>
```

Comment:
Is the styling weird broken here on purpose ? (i.e. the `Breadcrumb` text in a vertical line`

Reply:
oooo I cherry picked this commit from the layout PR as made more sense here and may not have checked it, will take another look, might be missing a class

Reply:
@philli-m  is this still pending?

Reply:
@m4ra @goapunk sorry forgot about that, fixed